nri-config-generator/main.go

```diff
@@ -139,3 +223,17 @@ func loadConfigTemplate() (*template.Template, error) {
 	}
 	return t, nil
 }
+
+// ProcessingRule is subset of the rules supported by nri-prometheus.
+type ProcessingRule struct {
```

Comment:
wouldn't the args package be a better fit for this types?

Reply:
makes sense, I'll change the args package name to `config`